argonaut/src/main/scala/io/finch/argonaut/Decoders.scala

```diff
@@ -8,8 +8,7 @@ import io.finch.internal.HttpContent
 
 trait Decoders {
 
-  /**
```

Comment:
Any chance we can undo these comment changes so it's easier to review?

Reply:
Or maybe open a separate PR with code style changes?

Reply:
yeah.  it was just the output of running `scalafmt` and `scalafix`